pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/impl/inv/OffHeapBitmapInvertedIndexCreator.java

```diff
@@ -71,6 +71,11 @@
   private static final Logger LOGGER =
       LoggerFactory.getLogger(OffHeapBitmapInvertedIndexCreator.class);
 
+  // Number of bytes required to store an integer
+  private static final int INT_SIZE = Integer.SIZE / Byte.SIZE;
+  // Maximum number of entries for each buffer
+  public static final int MAX_NUM_ENTRIES = Integer.MAX_VALUE / INT_SIZE;
```

Comment:
The actual size calculation is trickier. For multi-value columns, we allow this number of entries; for number of total docs, we allow this number of total docs. But we don't allow this high cardinality, which is what actually stored in the final bitmap file.